fe/fe-core/src/main/java/com/starrocks/qe/SimpleScheduler.java

```diff
@@ -47,38 +47,38 @@
 import com.starrocks.system.SystemInfoService;
 import com.starrocks.thrift.TNetworkAddress;
 import com.starrocks.thrift.TScanRangeLocation;
+import org.apache.arrow.util.VisibleForTesting;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicLong;
-import java.util.concurrent.locks.Lock;
-import java.util.concurrent.locks.ReentrantLock;
 import java.util.stream.Collectors;
 import javax.annotation.Nullable;
 
 public class SimpleScheduler {
-    private static Logger LOG = LogManager.getLogger(SimpleScheduler.class);
+    private static final Logger LOG = LogManager.getLogger(SimpleScheduler.class);
     //count id for compute node get TNetworkAddress
-    private static AtomicLong nextComputeNodeHostId = new AtomicLong(0);
+    private static final AtomicLong NEXT_COMPUTE_NODE_HOST_ID = new AtomicLong(0);
     //count id for backend get TNetworkAddress
-    private static AtomicLong nextBackendHostId = new AtomicLong(0);
+    private static final AtomicLong NEXT_BACKEND_HOST_ID = new AtomicLong(0);
+
     //count id for get ComputeNode
-    private static Map<Long, Integer> blacklistBackends = Maps.newHashMap();
-    private static Lock lock = new ReentrantLock();
-    private static UpdateBlacklistThread updateBlacklistThread;
-    private static AtomicBoolean enableUpdateBlacklistThread;
+    private static final ConcurrentMap<Long, Integer> BLACKLIST_BACKENDS = Maps.newConcurrentMap();
```

Comment:
can change to `BLOCKLIST` together with this refactor.

Reply:
Do you mean rename `BLACKLIST_BACKENDS` to `BLACKLIST`?

Reply:
literally repace `blacklist` with `blocklist`

Reply:
Got it.

Reply:
What's difference between blacklist and blocklist?